cli-feed-v3-3.json

```diff
@@ -0,0 +1,239 @@
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.win-x86.2.7.3188.zip",
+          "sha2": "4a0342497a606e763aa148a10852d6da91d2d5cc7fdb7e705a0c17df4e038023"
+        }
+      ]
+    },
+    "3.20.0": {
+      "cli": "https://functionscdn.azureedge.net/public/3.0.3284/Azure.Functions.Cli.min.win-x86.3.0.3284.zip",
+      "sha2": "a6367637da8ec1387e074dadc45f5da0dfceed876bb7d9140d5f17a16511bf3f",
+      "localEntryPoint": "func.exe",
+      "templateApiZip": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "extensionBundle": {
+        "hostConfig": {
+          "extensionBundle": {
+            "id": "Microsoft.Azure.Functions.ExtensionBundle",
+            "version": "[1.*, 2.0.0)"
+          }
+        }
+      },
+      "runtimeInformation": {
+        "dotnet": {
+          "net5-isolated": {
+            "displayInfo": {
+              "displayName": ".NET 5 (Isolated)",
+              "hidden": true,
+              "diplayVersion": "v3",
+              "targetFramework": ".NET 5"
+            },
+            "capabilites": "outOfProc",
+            "sdkName": "Microsoft.Azure.Functions.Worker.Sdk",
+            "sdkVersion": "1.0.0-preview5" ,
+            "itemTemplates": "https://ankikumapublic.blob.core.windows.net/templates/Microsoft.Azure.Functions.Worker.ItemTemplates.3.1.1705-preview1.nupkg",
+            "projectTemplates": "https://ankikumapublic.blob.core.windows.net/templates/Microsoft.Azure.Functions.Worker.ProjectTemplates.3.1.1705-preview1.nupkg",
+            "projectTemplateId": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.Isolated.3.x",
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/azure-functions/dotnet-isolated:3.0-dotnet-isolated5.0-appservice",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~3",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v5.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET-ISOLATED|5.0"
+            }
+          },
+          "netcore3-inproc": {
+            "displayInfo": {
+              "displayName": ".NET Core 3 (LTS)",
+              "hidden": false,
+              "diplayVersion": "v3",
+              "targetFramework": ".NET Core"
+            },
+            "capabilites": "inProc",
+            "sdkName": "Microsoft.NET.Sdk.Functions",
+            "sdkVersion": "3.0.1" ,
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ItemTemplates/3.1.1648",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/3.1.1648",
+            "projectTemplateId": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.3.x",
```

Comment:
This is a C# specific value; how about "projectTemplateId": { "csharp" : "the value" }? There are other dotnet languages that would have different templates.